src/metabase/api/collection.clj

```diff
@@ -815,33 +816,38 @@
         (for [model [:card :metric :dataset :dashboard :snippet :pulse :collection :timeline]]
           (:select (collection-children-query model {:id 1 :location "/"} nil)))))
 
+(defn- official-collections-first-sort-clause [{:keys [official-collections-first?]}]
```

Comment:
I think this key (`official-collections-first?`) doesn't match the one on https://github.com/metabase/metabase/pull/44322/files#diff-2bba2868bee09baf3be94bf21030bb7d7a1975aa36450ef6626601a09078d602R1003